libs/ngx-material-intl-tel-input-lib/src/lib/ngx-material-intl-tel-input-lib/ngx-material-intl-tel-input-lib.component.scss

```
@@ -131,10 +193,10 @@
       margin: 2px 0;
     }
     mat-hint {
-      color: #b2b2b2;
+      color: var(--mdc-tel-form-hint-color, #b2b2b2);
     }
     mat-error {
-      color: var(--mdc-theme-error, #f44336);
+      color: var(--mdc-tel-form-error-color, var(--mat-sys-error));
```

```suggestion
      color: var(--mdc-tel-form-error-color, var(--mat-sys-error, var(--mdc-theme-error, #f44336)));
```
Keep the previous values here as well.